migrations/sql/V2017.09.18.1058__TrackCurrentModelAndVersion.sql

```diff
@@ -0,0 +1,70 @@
+DELETE FROM model_version WHERE model = 'unknown';
+DELETE FROM model WHERE id = 'unknown';
+
+ALTER TABLE modelling_group
+    RENAME COLUMN current to replaced_by;
+
+ALTER TABLE model
+    DROP COLUMN current,
+    ADD COLUMN is_current boolean NOT NULL DEFAULT FALSE,
+    ADD COLUMN current_version serial,
```

Comment:
I don't think current-version should be a serial column; it references ids from the model_version table.